.github/workflows/release.yaml

```diff
@@ -44,15 +45,24 @@ jobs:
       - name: Install tool dependencies
         uses: jdx/mise-action@v2
 
-      - name: Package CRDs
+      - name: Package kubernetes manifests & CRDs
         run: |
           cat deploy/charts/prefect-operator/crds/*.yaml > prefect-crds.yaml
+          helm dependency build deploy/charts/prefect-operator
```

Comment:
I noticed in some other workflows that this command can fail if the `bitnami` repo hasn't been added (helm repo add bitnami ...) - if you didn't see that here then all good